src/app/onion/dashboard/dashboard.component.ts

```diff
@@ -213,7 +207,7 @@ export class DashboardComponent implements OnInit, OnDestroy {
    * @param text
    */
   performSearch(text: string) {
-    this.getReleasedLearningObjects({ status: LearningObject.Status.RELEASED }, text);
+    this.getReleasedLearningObjects({ status: LearningObject.Status.RELEASED, text });
```

Comment:
I thought the function expects two arguments, instead of a single object or is there a typescript magic I didn't know about?